configstack/module.go

```diff
@@ -577,7 +577,8 @@ func FindWhereWorkingDirIsIncluded(terragruntOptions *options.TerragruntOptions,
 		dir = dir + filepath.FromSlash("/")
 		cfgOptions, err := options.NewTerragruntOptions(dir)
 		if err != nil {
-			return nil, err
+			terragruntOptions.Logger.Debugf("Failed to build terragrunt options from %s %v", dir, err)
```

Comment:
This seems like a real error that should probably be returned?

Reply:
Since building list of all modules that use current module may not work in some cases(usage of relative paths) approach is to log as debug and ask user to confirm only detected dependencies

https://github.com/gruntwork-io/terragrunt/pull/1889

https://github.com/gruntwork-io/terragrunt/issues/299